operations/helm/charts/mimir-distributed/templates/metamonitoring/_helpers.tpl

```diff
@@ -56,3 +56,15 @@
     cluster: {{ include "mimir.clusterName" $.ctx | quote}}
 {{- end -}}
 {{- end -}}
+
+{{- define "mimir.metaMonitoring.metrics.remoteReadUrl" -}}
+{{- with $.ctx.Values.metaMonitoring.grafanaAgent.metrics }}
+{{- $writeBackToMimir := not (.remote).url -}}
+{{- if $writeBackToMimir -}}
+{{- include "mimir.remoteReadUrl.inCluster" $.ctx }}
+{{- else -}}
+{{- $parsed := urlParse (.remote).url -}}
+{{ $parsed.scheme }}://{{ $parsed.host }}{{ include "mimir.prometheusHttpPrefix" $.ctx }}
```

Comment:
For external URLs we need a sensible default like `/prometheus`  and allow an override somewhere, otherwise we're tying the internal setting and an unrelated external setting together. For an MVP, I'd just hardcode to `/prometheus` while we figure out where to put the override.

Reply:
nice spot. A Prometheus instance also doesn't have the `/prometheus` prefix, `/prometheus` is only a Mimir prefix.